### ISSUE TITLE
Linux: "Invalid maximum heap size: -Xmx128" with ./nzbhydra2

### ISSUE THREAD

**Issue by sanderjo** [theotherp/nzbhydra2#23]
I get the error below trying to start with `./nzbhydra2` (on Ubuntu 17.10 x86-64, openjdk version "1.8.0_151")

The strange thing: it **does** work with the exact command from `./nzbhydra2`, so 
```java -Xmx128M -DfromWrapper -XX:TieredStopAtLevel=1 -noverify -Dspring.output.ansi.enabled=ALWAYS -jar /home/sander/Downloads/nzbhydra2/hydra-1.0.5/lib/core-1.0.5-exec.jar --datafolder /home/sander/Downloads/nzbhydra2/hydra-1.0.5/data```

And it also works with a plain `java -jar lib/core-1.0.5-exec.jar directstart` 

So something strange because of or in ./nzbhydra2? Or something I'm doing wrong?

```
sander@sammie-1710:~/Downloads/nzbhydra2/hydra-1.0.5$ ./nzbhydra2
Logging wrapper output to /home/sander/Downloads/nzbhydra2/hydra-1.0.5/data/logs/wrapper.log
2018-01-06 23:33:34,422  INFO - Starting NZBHydra main process with command line: java -Xmx128
M -DfromWrapper -XX:TieredStopAtLevel=1 -noverify -Dspring.output.ansi.enabled=ALWAYS -jar /home/sander/Downloads/nzbhydra2/hydra-1.0.5/lib/core-1.0.5-exec.jar --datafolder /home/sander/Downloads/nzbhydra2/hydra-1.0.5/data in folder /home/sander/Downloads/nzbhydra2/hydra-1.0.5
Invalid maximum heap size: -Xmx128
M
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
2018-01-06 23:33:34,459  ERROR - Main process shut down unexpectedly. If the wrapper was started in daemon mode you might not see the error output. Start Hydra manually with the same parameters in the same environment to see it

sander@sammie-1710:~/Downloads/nzbhydra2/hydra-1.0.5$ 
```

**Comment by theotherp** [comment]
SHould be fixed with 1.0.6

**Comment by theotherp** [comment]
Should be fixed with 1.0.6

**Comment by sanderjo** [comment]
Indeed! Closing.
